src/credentials/BUILD.gn

```diff
@@ -62,6 +62,7 @@ static_library("credentials") {
   # For mbed and nrfconnect test builds, which are bilding monolithic test library these files are not needed.
```

Comment:
lets update this comment as well. Listing platforms here seems odd, so we can just say that `for some platforms` or similar and stop listing specific platform names.